### PR TITLE
added props types

### PR DESCRIPTION
**Pull request by shapikso**

### CODE REVIEW

File: src/components/Movies/Filters/DoubleRange/DoubleRange.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import React from 'react';
```
Comment (richerstars) [inline src/components/Movies/Filters/DoubleRange/DoubleRange.js:1]: different quotes

File: src/components/Movies/Filters/DoubleRange/DoubleRange.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-// import RangeInput from "./RangeInput";
-import InputValue from "./InputValue";
 import Slider from "rc-slider";
 import "./slider.scss";
+import Input from "../../../common/input/InputField";
+import PropTypes from "prop-types";
```
Comment (richerstars) [inline src/components/Movies/Filters/DoubleRange/DoubleRange.js]: Important import later then scss and slider

File: src/components/Movies/Filters/DoubleRange/DoubleRange.js [hunk 3]
```diff
@@ -25,5 +25,14 @@ const DoubleRange = ({title,minValue,maxValue, step, setRange, minInputValue, ma
...
 );
 
+DoubleRange.propTypes = {
+    minValue: PropTypes.string.isRequired,
+    maxValue: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    step: PropTypes.string.isRequired,
+    maxInputValue: PropTypes.number.isRequired,
+    minInputValue: PropTypes.number.isRequired,
+    setRange : PropTypes.func.isRequired
```
Comment (richerstars) [inline src/components/Movies/Filters/DoubleRange/DoubleRange.js:35]: no comma

File: src/components/Movies/Filters/FilterDate/FilterDate.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
```
Comment (richerstars) [inline src/components/Movies/Filters/FilterDate/FilterDate.js:1]: quotes as usually : or double, or once

File: src/components/Movies/Filters/FilterDate/FilterDate.js [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
+import PropTypes from "prop-types";
 
 const FilterDate = ({releaseDateFirst, releaseDateLast, setFilter}) => {
```
Comment (richerstars) [inline src/components/Movies/Filters/FilterDate/FilterDate.js:7]: no need new lane

File: src/components/Movies/Filters/FilterHeaders/FilterHeaders.js [hunk 6]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import Select from "../Selector/Select";
 import {movieLanguages, movieStatus} from "../../../../constants/selectors";
 import Input from "../../../common/input/InputField";
+import PropTypes from "prop-types";
```
Comment (nataliazhuravlova) [inline src/components/Movies/Filters/FilterHeaders/FilterHeaders.js]: libs imports at the top

File: src/components/Movies/Filters/FiltersRanges/FiltersRanges.js [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import DoubleRange from "../DoubleRange/DoubleRange";
-import PropTypes from "prop-types";
+import PropTypes from 'prop-types';
+import DoubleRange from '../DoubleRange/DoubleRange';
+
```
Comment (nataliazhuravlova) [inline src/components/Movies/Filters/FiltersRanges/FiltersRanges.js]: empty

File: src/components/Movies/Filters/Selector/Select.js [hunk 8]
```diff
@@ -1,13 +1,19 @@
 import React from 'react';
-import SelectItem from "./SelectItem";
+import PropTypes from 'prop-types';
+import SelectItem from './SelectItem';
 
-const Select = ({options,setDelector}) => {
-    const clickHandler = (e)=> setDelector(e.target.value);
+const Select = ({options,setSelector}) => {
+    const clickHandler = (e)=> setSelector(e.target.value);
```
Comment (nataliazhuravlova) [inline src/components/Movies/Filters/Selector/Select.js]: onChange

**Pull request merged**